lms/djangoapps/django_comment_client/utils.py

```diff
@@ -538,6 +561,25 @@ def get_ability(course_id, content, user):
 # TODO: RENAME
 
 
+def get_user_group_ids(course_id, content, user=None):
+    """
+    Given a user, course ID, and the content of the thread or comment, returns the group ID for the current user
+    and the user that posted the thread/comment.
+    """
+    content_user_group_id = None
+    user_group_id = None
+    if course_id is not None:
```

Comment:
It would be best to store the result of get_course_discussion_settings(course_id) here so that you don't make the call twice.